srtcore/buffer_rcv.cpp

```diff
@@ -591,14 +591,24 @@ int CRcvBuffer::getTimespan_ms() const
     if (m_iMaxPosInc == 0)
         return 0;
 
-    const int lastpos = incPos(m_iStartPos, m_iMaxPosInc - 1);
-    // Should not happen if TSBPD is enabled (reading out of order is not allowed).
-    SRT_ASSERT(m_entries[lastpos].pUnit != NULL);
+    int lastpos = incPos(m_iStartPos, m_iMaxPosInc - 1);
```

Comment:
What's the value of `m_iMaxPosInc` when the buffer is empty?

Reply:
Zero (`0`), but `m_iMaxPosInc` is checked to be positive one line above.

Reply:
Ah ok. Looks good then.